docs/design/w3c/spec.md

```diff
@@ -0,0 +1,530 @@
+issuer of a verifiable credential.
+
+In the case of W3C AnonCreds credentials, the `issuer` attribute should be represented as a
+resolvable [DID URL](https://w3c-ccg.github.io/did-resolution/) having either `indy` or `sov` DID method and looks the
+following:
+
+```
+{
+  ... 
+  "issuer": "did:sov:3avoBCqDMFHFaKUHug9s8W",
+  ...
+}
+```
+
+#### Issuance Date
+
+W3C [Issuance Date](https://www.w3.org/TR/vc-data-model/#issuance-date) section requires including of `issuanceDate`
+property to express the date and time when a credential becomes valid.
+
+In the case of W3C AnonCreds credentials, for the `issuanceDate` attribute recommended setting of a random time of the
+day when credential was issued or transformed and looks the following:
+
+```
+{
+  ... 
+  "issuanceDate": "2010-01-01T19:23:24Z",
+  ...
+}
+```
+
+#### Proofs (Signatures)
+
+W3C [Proofs (Signatures)](https://www.w3.org/TR/vc-data-model/#proofs-signatures) section requires including of `proof`
+property to express confirmation of the credential's validity.
+
+According to the specification, one or many proof objects can be added to verifiable credentials.
+In the context of W3C AnonCreds credentials included at least two proof object entries: AnonCreds CL
+and [Data Integrity](https://www.w3.org/TR/vc-data-model/#data-integrity-proofs).
+
+##### AnonCreds CL proof
+
+This proof entry derived from the CL signature of a verifiable credential.
+
+The defined `@context` includes a definition for the `CLSignature2022` type describing the format of the proof
+entry:
+
+```
+{
+  ... 
+  "proof": [
+    {
+      "type": "CLSignature2022",
+      "signature": "AAAgf9w5lZg....RYp8Z_x3FqdwRHoWruiF0FlM"
+    }
+  ]  
+  ...
+}
+```
+
+**Credential proof signature**
+
+* `type` - `CLSignature2022`
+* `signature` - credential signature
+    * signature received by building the following object from indy styled credential:
+      ```
+      {
+        "signature": {..}, 
+        "signature_correctness_proof": {..}
+      }
+      ```
+    * encoded
+      as [base64 attachment](https://github.com/hyperledger/aries-rfcs/tree/main/concepts/0017-attachments#base64url)
+
+
+* **TO DISCUSS**: Signature/Proof encoding: Which approach to use for encoding?
+    * Basic approach used in Aries attachments: Base 64 encoding of object serialized as JSON string
+    * Compact encoding implementing in Python PoC: Using the fact that most fields of credential signature and proof are big numbers
+
+##### Data Integrity proof
+
+In order to better conform to the W3C specification AnonCreds based credential also requires including
+of [Data Integrity Proof](https://www.w3.org/TR/vc-data-model/#data-integrity-proofs) which must be generated using one
+of NIST-approved algorithms (RSA, ECDSA, EdDSA).
+
+Including Data Integrity proof allows to use verifiable credential without access to a Ledger.
+
+#### Expiration
```

Comment:
I don't we can map this on a specification level between an anoncreds credential and w3c credential as anoncreds does not have this concept. In that case, I'd say that we should be strict that the anoncreds credential will not work with the `expirationDate` property.

Reply:
Agree. As with the `credentialSubject: id`, we might sometime later add metadata such that the issuer can indicate that `attribute X contains the expiry date`, but that adds a bunch of uncertainty about date format, etc.  Not for now, I don’t think.